features/bevragen/persoon/verblijfplaats/overzicht.feature

```diff
@@ -0,0 +1,95 @@
+    Gegeven de persoon met burgerservicenummer '000000097' heeft de volgende 'verblijfplaats' gegevens
+    | naam                               | waarde   |
+    | datum aanvang adreshouding (10.30) | 20170423 |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                 | waarde   |
+    | gemeentecode (92.10) | 0518     |
+    | <gba veld>           | <waarde> |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000097                       |
+    | fields              | verblijfplaats.type             |
+    Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+    | naam | waarde |
+    | type | Adres  |
+
+    Voorbeelden:
+    | gba veld                                   | waarde           |
+    | straatnaam (11.10)                         | Spui             |
+    | naam openbare ruimte (11.15)               | Spui             |
+    | huisnummer (11.20)                         | 123              |
+    | huisletter (11.30)                         | a                |
+    | huisnummertoevoeging (11.40)               | 2                |
+    | aanduiding bij huisnummer (11.50)          | to               |
+    | postcode (11.60)                           | 1234AA           |
+    | woonplaatsnaam (11.70)                     | 's-Gravenhage    |
+    | identificatiecode verblijfplaats (11.80)   | 0599010000208579 |
+    | identificatiecode nummeraanduiding (11.90) | 0599200000219678 |
+
+Rule: Een verblijfplaats is een binnenlandse locatie wanneer locatiebeschrijving (12.10) een waarde heeft
+
+  Scenario: 'locatiebeschrijving (12.10)' heeft een waarde 
+    Gegeven de persoon met burgerservicenummer '000000097' heeft de volgende 'verblijfplaats' gegevens
+    | naam                               | waarde   |
+    | datum aanvang adreshouding (10.30) | 20170423 |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                        | waarde                       |
+    | gemeentecode (92.10)        | 0518                         |
+    | locatiebeschrijving (12.10) | Woonboot in de Grotere Sloot |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000097                       |
+    | fields              | verblijfplaats.type             |
+    Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+    | naam | waarde  |
+    | type | Locatie |
+
+Rule: Een verblijfplaats is een verblijfplaats buitenland wanneer tenminste één gegeven van adres buitenland (groep 13) een waarde heeft. De waarde van 'land adres buitenland (13.10)' is ongelijk aan 0000
```

Comment:
In 'informatievragen-fields-gba.feature' wordt aangegeven dat als verblijfplaats (of een van de velden daarvan) wordt gevraagd dat alle beschikbare verblijfplaatsvelden worden geleverd met de volgende kanttekening:

`dit betreft alle velden uit groepen 11 (adres), 12 (locatie) en land, regel 1, regel 2 en regel 3 van groep 13 (verblijf buitenland)`

Dat betekent dat je op basis van alleen 'datum aanvang adres buitenland (13.20)' niet kunt vaststellen dat het type 'VerblijfplaatsBuitenland' is. Of deze kanttekening is niet correct of bovenstaande rul.

Reply:
volgens mij moet land ongelijk zijn aan 0000 en is het niet relevant of regel 1, 2 en 3 een (standaard)waarde heeft